specs/monitor_build.spec

```
@@ -115,10 +127,10 @@ exe = EXE(
     runtime_tmpdir=None,
     console=True,
     disable_windowed_traceback=False,
-    argv_emulation=False,
-    target_arch=None,
+    argv_emulation=True if sys.platform == 'darwin' else False,  # macOS 需要开启
+    target_arch='arm64' if sys.platform == 'darwin' else None,  # Apple Silicon 架构
     codesign_identity=None,
     entitlements_file=None,
-    icon='assets/icon.ico' if os.path.exists('assets/icon.ico') else None,
+    icon='assets/icon.ico' if sys.platform == 'win32' else None,  # macOS 暂不使用图标
```

RuboCop (1.81.7)

[fatal] unexpected token kELSE
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)